cli/version_check.go

```diff
@@ -25,6 +26,13 @@ func PopulateTerraformVersion(terragruntOptions *options.TerragruntOptions) erro
 	terragruntOptionsCopy.Writer = ioutil.Discard
 	terragruntOptionsCopy.ErrWriter = ioutil.Discard
 
+	// Remove any TF_CLI_ARGS before version checking
```

Comment:
NIT: could you expand this comment to explain why we're removing these (i.e., these are args to be appended to the "real" command and not the `--version` check) and add a link to the `TF_CLI_ARGS` docs?

Reply:
Sure! Updated.